### PR TITLE
set touched when dropdown, forces to validate

### PR DESCRIPTION
**Pull request by salomegeo**
## Description
968

## Change Type

- Bug Fix

## Code Checklist
- [ ] Code compiles successfully (verified via `yarn start`)
- [ ] No lint issues exist (verified via `yarn lint`)
- [ ] New and existing unit tests pass (verified via `yarn test`)
- [ ] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/SendBch/FirstStep/index.js [hunk 1]
```diff
@@ -9,6 +10,16 @@ import Loading from './template.loading'
 import Success from './template.success'
 
 class FirstStep extends React.Component {
+  constructor (props) {
+    super(props)
+    this.handleToToggle = this.handleToToggle.bind(this)
+  }
+
+  handleToToggle (val) {
+    this.props.dispatch(touch('sendBch', 'to'))
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SendBch/FirstStep/index.js]: `this.props.formActions.touch('sendBch', 'to')`

File: packages/blockchain-wallet-v4-frontend/src/modals/SendBch/FirstStep/index.js [hunk 2]
```diff
@@ -35,7 +46,8 @@ const mapStateToProps = state => ({
 })
 
 const mapDispatchToProps = dispatch => ({
-  actions: bindActionCreators(actions.components.sendBch, dispatch)
+  actions: bindActionCreators(actions.components.sendBch, dispatch),
+  dispatch: dispatch
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SendBch/FirstStep/index.js]: `formActions: bindActionCreators(actions.form, dispatch)`

File: packages/blockchain-wallet-v4-frontend/src/modals/SendBch/FirstStep/index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
+import { touch } from 'redux-form'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SendBch/FirstStep/index.js]: remove

**Comment by salomegeo** [comment]
READY TO BE APPROVED!!!!

**Pull request merged**